### PR TITLE
V1.1.0

### PR DESCRIPTION
**Pull request by ljwh1231**

### CODE REVIEW

File: constant.go [hunk 1]
```diff
@@ -6,11 +6,67 @@ import (
...
+		"081": "하나은행",
+		"088": "신한은행",
+		"089": "케이뱅크",
+		"090": "카카오뱅크",
+		"012": "농협중앙회",
+		"045": "새마을금고중앙회",
+		"048": "신협중앙회",
+		"064": "산림조합중앙회",
+		"071": "우체국은행",
+	}
```
Comment (kim-dali) [inline constant.go]: struct

File: mock_payletter.go [hunk 2]
```diff
@@ -97,3 +98,66 @@ func (o *MockPayLetter) CancelTransaction(req ReqCancelTransaction) (res ResCanc
 	}
 	return
 }
+
+func (o *MockPayLetter) RegisterEasyPay(req ReqRegisterEasyPay) (res *ResRegisterEasyPay, err error) {
```
Comment (kim-dali) [inline mock_payletter.go]: 포인터 아니여도 될 것 같아요!

File: type.go [hunk 3]
```diff
@@ -10,6 +10,7 @@ import (
 
 type pgCode struct {
 	CreditCard string
+	Easybank   string
```
Comment (kim-dali) [inline type.go]: constant 로 옮겨 주세용

File: type.go [hunk 4]
```diff
@@ -143,3 +148,63 @@ type ResCancelTransaction struct {
...
+	CancelUrl     string `json:"cancel_url"`
+	ReqDate       string `json:"req_date"`
+	HashData      string `json:"hash_data"`
+}
+
+type ResRegisterEasyPay struct {
+	Token       *string
+	RedirectUrl *string `json:"redirect_url"`
+	Code        *int
+	Message     string
```
Comment (kim-dali) [inline type.go]: json tag

File: type.go [hunk 5]
```diff
@@ -143,3 +148,63 @@ type ResCancelTransaction struct {
...
+
+type PayletterMethod struct {
+	PaymentMethod         string `json:"payment_method"`
+	Billkey               string `json:"billkey"`
+	AliasName             string `json:"alias_name"`
+	FavoriteFlag          string `json:"favorite_flag"`
+	MethodRegDate         string `json:"method_reg_date"`
+	MethodInfo            string `json:"method_info"`
+	MethodCode            string `json:"method_code"`
+	MethodName            string
```
Comment (kim-dali) [inline type.go]: prettier 개행

File: constant.go (hunk 1 above)
Comment (ljwh1231) [reply constant.go]: [24bb2aa](https://github.com/whitecubeinc/payletter/pull/1/commits/24bb2aa6433a22e884e8c5349c8887770383a6f1)

File: type.go (hunk 3 above)
Comment (ljwh1231) [reply type.go]: [24bb2aa](https://github.com/whitecubeinc/payletter/pull/1/commits/24bb2aa6433a22e884e8c5349c8887770383a6f1)

File: mock_payletter.go (hunk 2 above)
Comment (ljwh1231) [reply mock_payletter.go]: [df96a32](https://github.com/whitecubeinc/payletter/pull/1/commits/df96a32a6a83c2e59ce2b27135b4eb8675e9b0e6)

File: type.go (hunk 4 above)
Comment (ljwh1231) [reply type.go]: [42abc5d](https://github.com/whitecubeinc/payletter/pull/1/commits/42abc5dbee1133283d404c20d5b50c60dbd6ef6d)

File: type.go (hunk 5 above)
Comment (ljwh1231) [reply type.go]: [42abc5d](https://github.com/whitecubeinc/payletter/pull/1/commits/42abc5dbee1133283d404c20d5b50c60dbd6ef6d)

**Pull request merged**